src/main/java/kr/motd/maven/os/Detector.java

```diff
@@ -143,4 +159,74 @@ private static String normalize(String value) {
         }
         return value.toLowerCase(Locale.US).replaceAll("[^a-z0-9]+", "");
     }
+
+    /**
+     * Gets a collection of OSes that the given detected OS is "like". The {@code detectedName} will
+     * always appear in the collection.
+     */
+    private static Collection<String> getOsLike(String detectedName) {
+        String[] osReleaseFileNames = {"/etc/os-release", "/usr/lib/os-release"};
+        Set<String> like = new LinkedHashSet<String>();
+        for (String osReleaseFileName : osReleaseFileNames) {
+            File file = new File(osReleaseFileName);
+            if (file.exists()) {
+                like.addAll(parseLinuxOsReleaseFile(file));
+                break;
+            }
+        }
+
+        // Older versions of redhat don't have /etc/os-release. In this case, just
+        // add "rhel" and "fedora" to the list.
+        if (like.isEmpty() && isRedHatVariant()) {
```

Comment:
Only do this if the above failed? Fedora will have /etc/redhat-release even though it is not "like" rhel.